checker/worker/metrics.go

```diff
@@ -61,7 +61,7 @@ func (worker *Checker) addRemoteTriggerIDsIfNeeded(triggerIDs []string) {
 
 func (worker *Checker) getTriggerIDsToCheck(triggerIDs []string) []string {
 	lazyTriggerIDs := worker.lazyTriggerIDs.Load().(map[string]bool)
-	triggerIDsToCheck := make([]string, len(triggerIDs))
+	var triggerIDsToCheck []string
```

Comment:
hm, why?

Reply:
Each empty cell produce trigger processing, that of course will be prevented on some step, but this produces log records those make a mess in logs

Reply:
It is strange that there was instantiating of slice with len of triggers and then appending IDs to it, as a result we have slice with N empty strings and M triggers IDs to check (M <= N). @maksgalimz Is it was just a bug?

Reply:
```suggestion
	triggerIDsToCheck := make([]string, 0, len(triggerIDs))
```

Reply:
Maybe author wanted to set capacity of slice

Reply:
Agree